manifests/k/KandoMenu/Kando/1.8.0/KandoMenu.Kando.installer.yaml

```diff
@@ -0,0 +1,21 @@
+# Created with WinGet Releaser using komac v2.11.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: KandoMenu.Kando
+PackageVersion: 1.8.0
+InstallerType: exe
+InstallModes:
+- silent
+InstallerSwitches:
+  Silent: --silent
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
+ReleaseDate: 2025-03-27
+Installers:
+- Architecture: x86

```

Comment:
```suggestion
- Architecture: x64
```

Reply:
Thanks for pointing this out. Seems like winget-releaser picked the wrong architecture here...